server/integration_tests/test_data/e2e_no_default_place_specified/chart_config.json

```diff
@@ -327,65 +327,6 @@
             "denom": "Count_Person",
             "title": "Population: Female, Enrolled in School in States of United States of America"
           },
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "Count_Death_Female_AsAFractionOf_Count_Person_Female"
```

Comment:
Is this a good thing we lost this?

Reply:
the query is "female population" & since this information about the female population got replace by other information about the female population, this is ok?

Reply:
yup, agreed.